server/resolvers/testPlanReportsResolver.js

```diff
@@ -10,12 +10,11 @@ const {
 
 const testPlanReportsResolver = async (
     _,
-    { statuses, testPlanVersionId, testPlanVersionIds, atId },
+    { phases = [], testPlanVersionId, testPlanVersionIds, atId, isApproved },
     context,
     info
 ) => {
```

Comment:
I ran this resolver in GraphQL and when I specified phases, I got an error.

I ran this query:

```
query {
  testPlanReports(phases: [RD]) {
    id
  }
}
```

Here was the error I got:

```
            "TypeError: Cannot read properties of undefined (reading 'phase')",
            "    at /Users/Shared/Projects/aria-at-app/server/resolvers/testPlanReportsResolver.js:68:60",
            "    at Array.filter (<anonymous>)",
            "    at Object.testPlanReportsResolver [as testPlanReports] (/Users/Shared/Projects/aria-at-app/server/resolvers/testPlanReportsResolver.js:67:32)",
            "    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)"
```

This happened no matter the phase I specified. I was using the test data.

Reply:
This should now work as expected in https://github.com/w3c/aria-at-app/pull/715/commits/b92d12dfebd00954c11d29638dafae6d889f0468

Reply:
Verified, thanks.